src/lib/rpc/rpc.ts

```diff
@@ -31,8 +31,13 @@ export async function invoke_rpc<T>(
     } else {
       exceptionLog(`Error calling '${cmd}'`, e);
     }
-    const toastMessage = toastOnError ?? "An unexpected error occurred";
-    toastStore.makeToast(toastMessage, "error");
+    // TODO - this is a dumb hack but whatever for now
+    if (toastOnError === "_mirror_") {
```

Comment:
I had some back and forth in my head when adding this wrapper about if it was even worth hiding the error. An alternative could be to instead of a "nice" `"An unexpected error occurred"` if no override is provided, just always leak the underlying error.

This magic string approach seems fine as well, but I think the effect could be made more clear with something glike "_show_invoke_error_".

Reply:
Yeah that's a good way to go, maybe even better to do both:
```
An unexpected error occurred <-- can be translated
<the underlying error>
```
can be refactored in the near future.